client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/steps/step-install-migation-plugin.tsx

```diff
@@ -2,14 +2,14 @@ import config from '@automattic/calypso-config';
 import { ExternalLink } from '@automattic/components';
 import { useTranslate } from 'i18n-calypso';
 import { recordMigrationInstructionsLinkClick } from '../tracking';
-import { getPluginInstallationPage } from './utils';
+import { getMigrationPluginInstallURL } from './utils';
 import type { FC } from 'react';
 
 interface Props {
 	fromUrl: string;
 }
 
-export const StepInstallMigrationGuru: FC< Props > = ( { fromUrl } ) => {
+export const StepInstallMigrationPlugin: FC< Props > = ( { fromUrl } ) => {
```

Comment:
Can we please fix the typo in the file name too (i.e. `step-install-migation-plugin.tsx` -> `step-install-migration-plugin.tsx`)?

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/95153/commits/9bd3e5c0d99f6f28805bc305a20ada97531146ca. 👍